store/tikv/kv/error.go

```diff
@@ -49,6 +51,8 @@ var (
 	ErrTokenLimit                  = dbterror.ClassTiKV.NewStd(mysql.ErrTiKVStoreLimit)
 	ErrLockExpire                  = dbterror.ClassTiKV.NewStd(mysql.ErrLockExpire)
 	ErrUnknown                     = dbterror.ClassTiKV.NewStd(mysql.ErrUnknown)
+
+	ErrNotExist = NewError(mysql.ErrCodeNotExist)
```

Comment:
Will the other errors above be replaced with errors defined like this?

Reply:
Yes